src/component/feature/feature-tag-component.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Chip, ChipContact } from 'react-mdl';
+import AddTagDialog from './add-tag-dialog-container';
+
+function FeatureTagComponent(props) {
+    const untagFeature = tagId => {
+        // eslint-disable-next-line no-alert
+        if (window.confirm('Are you sure you want to remove this tag')) {
+            props.untagFeature(props.featureToggleName, tagId);
+        }
+    };
+
+    const tagIcon = typeName => {
+        let tagType = props.tagTypes.find(type => type.name === typeName);
+        if (tagType) {
+            if (tagType.icon.indexOf('https://') === 0) {
```

Comment:
tagType.icon can be empty.

Reply:
I'm not sure if we ever will allow the icon to be an actual url. E.g. for the hosted offering we have strict CSP headers to avoid including from unknown domains.